### DISCUSSION TITLE
Enabling cookie consent

### DISCUSSION THREAD

**Discussion by pav-anastasiadis** [mmistakes/jekyll-theme-basically-basic#140] [Q&A]
A guide to create or simplify the process of adding a banner for cookie consent would be greatly appreciated.  I've discovered a guide for a similar implementation on minimal-mistakes [here](https://ptc-it.de/enabling-cookie-consent-with-jekyll-minimal-mistakes/) but unfortunately it doesn't translate well for the current manner google analytics are set up.

**Comment by henrieglesorotos** [comment]
Agree - I'd quite like to add GA to my site, but won't be possible without a cookie banner. Having something out of the box would be far more easy.
